packages/api/src/utils/psa.js

```diff
@@ -41,7 +41,7 @@ export const getEffectivePinStatus = (pins) => {
 
 const psaStatusesToDBStatusesMap = {
   pinned: ['Pinned'],
-  queued: ['PinQueued'],
+  queued: ['PinQueued', 'Unpinned', 'Remote'],
```

Comment:
Interesting. So, `Remote` will always be remote and not change https://github.com/web3-storage/web3.storage/blob/api-v5.4.1/packages/api/src/utils/psa.js#L28 Either when it is in queue for other node, or or when the other node actually pinned the content.

So I think we should filter out Remote status completely. Also consider we want to get rid of tracking these https://github.com/web3-storage/web3.storage/issues/1098

Reply:
So, from your debugging, you suspect Unpinned is the root of this issue?

Reply:
> So I think we should filter out Remote status completely. Also consider we want to get rid of tracking these https://github.com/web3-storage/web3.storage/issues/1098

Ok, I thought that Remote status was a transient state. 
(Are these statuses documented somewhere, we should add a link to that documentation if that's the case).

So assuming 3 nodes in the cluster
`Remote Remote Remote` --> `failed`
`Remote PinError PinError` --> `failed`
etc

If that is the case we should get rid of it from [here](https://github.com/web3-storage/web3.storage/blob/api-v5.4.1/packages/api/src/utils/psa.js#L28) and [here](https://github.com/web3-storage/web3.storage/pull/1110/files/16cddf8f1a92a4ffc3a302f25fd7e827cb1c8a9b#diff-da2c8b3756f3bdc198b20d150e214a5ef6fd6d43b851f1707b189a7b37b9a274R44), isn't?

> So, from your debugging, you suspect Unpinned is the root of this issue?

Not having `Unpinned` here is defo a problem given, the cluster may return "Unpinned" as the initial status.
That results in having that specific PSA request to return `queued` when hitting GET pin/{id} directly (because of [this logic](https://github.com/web3-storage/web3.storage/blob/api-v5.4.1/packages/api/src/utils/psa.js#L32)) but not appear in the list when filtering by `queued`. When I checked the pins for the request Lidel reported all the pins were `Unpinned` indeed.

Reply:
@flea89 could only find docs in https://cluster.ipfs.io/documentation/guides/pinning/#filtering-results

> If that is the case we should get rid of it from [here](https://github.com/web3-storage/web3.storage/blob/api-v5.4.1/packages/api/src/utils/psa.js#L28) and [here](https://github.com/web3-storage/web3.storage/pull/1110/files/16cddf8f1a92a4ffc3a302f25fd7e827cb1c8a9b#diff-da2c8b3756f3bdc198b20d150e214a5ef6fd6d43b851f1707b189a7b37b9a274R44), isn't?

Yeah, I think so.

Reply:
Removed.

I think we should get this in and I'll create a separate issue for the issue I described [here](https://github.com/web3-storage/web3.storage/pull/1110#issue-1168108666).